deploy/default.conf

```diff
@@ -190,10 +190,10 @@ http {
 
     location / {
         absolute_redirect off;
-        proxy_connect_timeout 90;
-        proxy_send_timeout 90;
-        proxy_read_timeout 90;
-        send_timeout 90;
+        proxy_connect_timeout 300;
+        proxy_send_timeout 300;
+        proxy_read_timeout 300;
+        send_timeout 300;
```

Comment:
Changes made as recommended by the developers: https://label-studio.slack.com/archives/CQ8LYPPJS/p1691861521592959?thread_ts=1691700803.437549&cid=CQ8LYPPJS